src/components/Footer/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Footer';
```

Comment:
Consider importing the Footer module and then exporting it separately, rather than using `export *`.